index.js

```diff
@@ -109,6 +109,18 @@ const allEditors = () => [
 			'/Applications/Emacs.app/Contents/MacOS/Emacs'
 		],
 		keywords: []
+	},
+	{
+		id: 'androidstudio',
```

Comment:
```suggestion
		id: 'android-studio',
```

I do realize I haven't been consistent about this.

Reply:
Done.